app/AndroidManifest.xml

```diff
@@ -67,9 +67,8 @@
                 <action android:name="android.intent.action.VIEW"/>
                 <category android:name="android.intent.category.DEFAULT"/>
                 <category android:name="android.intent.category.BROWSABLE"/>
-                <data android:scheme="https"
-                  android:host="appauth.demo-app.io"
-                  android:path="/oauth2redirect"/>
+                <data android:scheme="appscheme"
```

Comment:
Hello Iain, Yes I have changed scheme here. Are you recommending this should not changed here?